script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -942,7 +942,7 @@ I don't care if what she says is true...
 [sha:8c69d20f2a69d21e259d09f1bc9caabba57e0d32]{
 -- Page 188, Offset 33721.
 -- ……頭が痺れて何も考えられないのは。
-It's something else that's numbed my brain and robbed me of the ability to think:
+It's something else that's numbed my brain and robbed me of the ability to think.
```

Comment:
Links weirdly now

Might have to change the next line if you want to keep this